src/index-debug.html

```diff
@@ -24,38 +24,17 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Photoshop Design Space (optimized build)</title>
-    <link rel="stylesheet" type="text/css" href="style/main.css" />
-    <script src="js/require.js"></script>
-    <script>
-        var locale = window.navigator.language,
-            parts = locale.match(/\w+/),
-            language = parts.length > 0 && parts[0];
+    <title>Photoshop Design Space (development build)</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type ="text/javascript" src="./externalDeps.js"></script>
```

Comment:
Well, it does seem like there is a minor performance regression here, but I guess it isn't as bad as I first thought, since it seems like with the r.js-compiled sources it also takes 850ms or so before we start running our code:
![image](https://cloud.githubusercontent.com/assets/1075154/11513459/204cbe82-9828-11e5-80c6-4e72c5ff5fb8.png)